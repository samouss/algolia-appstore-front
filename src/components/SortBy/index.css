.SortBy__Label {
  padding: 5px 10px;
}
