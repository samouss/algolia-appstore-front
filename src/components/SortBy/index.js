import React from 'react';
import PropTypes from 'prop-types';
import './index.css';

const SortBy = ({ children }) => (
  <div>
    <p styleName="SortBy__Label">
      Sort by:
    </p>
    {children}
  </div>
);

SortBy.propTypes = {
  children: PropTypes.node.isRequired,
};

export default SortBy;
