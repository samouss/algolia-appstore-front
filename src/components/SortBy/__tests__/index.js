import React from 'react';
import { shallow } from 'enzyme';
import SortBy from '../index';

describe('<SortBy />', () => {
  const defaultProps = {};

  it('expect to render', () => {
    const props = {
      ...defaultProps,
    };

    const component = shallow(
      <SortBy
        {...props}
      >
        Content
      </SortBy>,
    );

    expect(component).toMatchSnapshot();
  });
});
