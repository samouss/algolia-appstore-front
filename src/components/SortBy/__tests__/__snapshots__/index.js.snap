// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SortBy /> expect to render 1`] = `
<div>
  <p
    styleName="SortBy__Label"
  >
    Sort by:
  </p>
  Content
</div>
`;
